getting-started-command-mode/pom.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project>
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.acme</groupId>
+    <artifactId>getting-started-command-mode</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <quarkus-plugin.version>999-SNAPSHOT</quarkus-plugin.version>
```

Comment:
it has to be 1.4.0.Final or 1.4.1.Final, right? not 999-SNAPSHOT

Reply:
not in development branch.

See checklist when opening a pr:
- [ ] uses the `999-SNAPSHOT` version of Quarkus

if you are asking in general - then yes, this feature was first released in 1.4.0.Final.